atomicapp/requirements.py

```diff
@@ -79,12 +79,12 @@ def _exec(self, action):
             requirement_function = self._find_requirement_function_name(key_name)
 
             # Check to see if the function exists in the provider,
-            # if it does not: fail
+            # if it does not: warn the user
             try:
                 requirement = getattr(self.provider, requirement_function)
             except AttributeError:
-                raise RequirementFailedException(
-                    "Requirement %s does not exist within %s." %
+                return logger.warning(
+                    "Requirement %s does not exist within %s. Skipping." %
                     (requirement_function, self.provider))
```

Comment:
hmm. this is misleading. we should really be doing:
```
logger.warning()
return
```

otherwise it is misleading because new users might think the return value is actually being used somewhere.